fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/predicates/QuantifiedColumnValue.java

```diff
@@ -130,6 +153,11 @@ public boolean isFunctionallyDependentOn(@Nonnull final Value otherValue) {
 
     @Nonnull
     public static QuantifiedColumnValue of(@Nonnull CorrelationIdentifier alias, int ordinal) {
-        return new QuantifiedColumnValue(alias, ordinal);
+        return new QuantifiedColumnValue(alias, ordinal, Type.Record.erased());
```

Comment:
Can you add a TODO for this one?

Reply:
I think you mean we should be able to construct the record type information, right?